process-migration-service/src/test/resources/kjars/test-1.0.0.pom

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.myspace</groupId>
+  <artifactId>test</artifactId>
+  <version>1.0.0</version>
+  <packaging>kjar</packaging>
+  <name>test</name>
+  <description></description>
+  <dependencies>
+    <dependency>
```

Comment:
I don't think we need any of the dependencies here so feel free to remove them. Same goes for kie-maven-plugin as the jar is already build. So remove that too and change packaging to jar - that way we get rid of all references for versions which could be problematic on some environments.